pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -260,6 +268,21 @@ func NewFramework(r Registry, plugins *config.Plugins, args []config.PluginConfi
 	return f, nil
 }
 
+func getPluginArgsOrDefault(pluginConfig map[string]runtime.Object, name string) (runtime.Object, error) {
```

Comment:
document this can return `nil, nil` if the plugin is out of tree or does not have a defined arg type

Reply:
Done